libgobuster/statehelpers.go

```diff
@@ -19,7 +19,7 @@ package libgobuster
 import (
 	"crypto/tls"
 	"fmt"
-	"github.com/hashicorp/go-multierror"
+	"go-multierror"
```

Comment:
Why this change? It's not part of stdlib?

Reply:
honestly I have no idea how this happened but it breaks the build.. oops :>